### ISSUE TITLE
When using keyboard navigation the child grid does not scroll to next row when next child is empty.

### ISSUE THREAD

**Issue by skrustev** [IgniteUI/igniteui-angular#4153]
## Description  
When using keyboard navigation and trying to navigate through a child grid that has not data for its child grid the newly selected cell is not visible.

 * igniteui-angular version: 7.2.x
 * browser: All

## Steps to reproduce  

1. Open the Hierarchical Grid Updating demo
2. Expand the first row
3. Expand the first row of the child grid
4. Click on the first cell in the child grid.
5. Press Arrow down

## Result  
The child grid is not scrolled to the selected cell.

## Expected result  
The child grid should scroll to the selected cell.

## Attachments  
Attach a sample if available, and screenshots, if applicable.
